src/server/router.js

```diff
@@ -16,5 +22,12 @@ router.getAsync('/products/discounts/async', sales.async);
 router.postAsync('/upload', uploads.writeAsyncInFile);
 router.getAsync('/upload/filenames', uploads.filenames);
 router.postAsync('/upload/optimization/:filename', uploads.optimization);
+router.postAsync('/upload/toDB', uploads.writeAsyncInDB);
+
+router.postAsync('/products/add', products.createProduct);
+router.getAsync('/products/get/:id', products.getProduct);
+router.putAsync('/products/update/:id', products.updateProduct);
+router.deleteAsync('/products/delete/:id', products.deleteProduct);
+router.getAsync('/products/getAll', products.getAllProducts);
```

Comment:
Single path can be used to cover all these cases.
Example:
GET `/products/` - returns all products
GET `/products/:id` - returns a product with a certain id
DELETE `/products/:id` - removes a product
POST `/products/` - adds a product
UPDATE `/products/` - updates a product

https://www.restapitutorial.com/lessons/httpmethods.html

And also. Why do you use POST to create a product and PUT to update a product?

Reply:
Because POST in my opinion created for **create** something on server. And PUT to update. But instead PUT I will be using PATCH method it has more valuable.